src/ui/server/helpers/task-list/index.ts

```diff
@@ -42,7 +42,7 @@ export const generateSimplifiedTaskList = (taskList: TaskListData): Array<TaskLi
           status: task.status,
           title: task.title,
         })),
-      } as TaskListGroup),
+      }) as TaskListGroup,
```

Comment:
my linting normally throws an error for this if its not all in the bracket

Reply:
Not sure why as we all should have same linting. Are you using VS code linting on top of prettier?

Reply:
I think this is now enforced with new version of prettier so without an `npm install` the linting would be different?